doc/classes/HTTPClient.xml

```diff
@@ -4,8 +4,10 @@
 		Hyper-text transfer protocol client.
 	</brief_description>
 	<description>
-		Hyper-text transfer protocol client. Supports SSL and SSL server certificate verification.
-		Can be reused to connect to different hosts and make many requests.
+		Hyper-text transfer protocol client (sometimes called "User Agent"). Usable to make HTTP requests to download web content, upload files and other data or to communicate with various services, among other use cases.
+		Note that this client only needs to connect to a host once (see [method connect_to_host]) to send multiple requests, and that because of this, methods that take URLs usually expect just the part after the host, not the full URL, as the client is already connected to a host (see [method request] for a full example and to get started).
```

Comment:
remove comma: "requests, and"
Actually overall this is really a run-on sentence and should be split into 2 or 3.